frontend/src/pages/MetadataByMunicipality/index.jsx

```diff
@@ -110,6 +124,7 @@ const MetadataByMunicipality = () => {
           setFetchedMunicipalities(receivedMunicipalities);
           setMunicipalities(receivedMunicipalities);
         }
+        if (name) setSelectedMunicipality(name);
```

Comment:
Not a big fan of if statements without curly brackets. Also: I am thoroughly confused as to why this whole thing is a useState, looks more like something which should be in a useEffect?